iib/workers/tasks/build_merge_index_image.py

```diff
@@ -285,3 +288,58 @@ def handle_merge_request(
     set_request_state(
         request_id, 'complete', 'The index image was successfully cleaned and updated.'
     )
+
+
+def is_bundle_version_valid(bundle_path, valid_ocp_version):
+    """
+    Check if the version label of the bundle satisfies the index ocp_version.
+
+    :param str bundle_path: pull specification of the bundle to be validated.
+    :param str valid_ocp_version: the index ocp version against which the bundles will be validated.
+    :return: a boolean indicating if the bundle_path satisfies the index ocp_version
+    :rtype: bool
+
+           |  "v4.5"   |   "=v4.6"    | "v4.5-v4.7" | "v4.5,v4.6"
+    ---------------------------------------------------------------
+    v4.5   | included  | NOT included |  included   |  included
+    ---------------------------------------------------------------
+    v4.6   | included  |   included   |  included   |  included
+    ---------------------------------------------------------------
+    v4.7   | included  | NOT included |  included   |  included
+    """
+    try:
+        float_valid_ocp_version = float(valid_ocp_version.replace('v', ''))
+    except ValueError:
+        raise IIBError(f'Invalid ocp_version "{valid_ocp_version}" specified for target_index')
```

Comment:
Within the scope of this function `target_index` and `ocp_version` are unknown variables. Can we tweak the terminology to say something like:
> Invalid OCP version, "vYolo", specified in Index Image.

If knowing which Index Image is important, you can log a message before `is_bundle_version_valid` is called to give additional context.

Reply:
Done